### ISSUE TITLE
[generate:plugin:block] Options can not be generated in no-interaction mode

### ISSUE THREAD

**Issue by viveksr2** [hechoendrupal/drupal-console#3975]
### Problem/Motivation
When generating block with fields like checkboxes, radio or combo, options need to be passed to the command. However, as the field info is separated by comma(,) thus the options get separated in-appropriately, thus broken code is generated.

**Details to include:**
- The extraction of input parameters is done in ArrayInputTrait inside explodeInlineArray() method.

### How to reproduce
Run following command and observe the form definition in the generated code.

drupal --no-interaction generate:plugin:block --module=TestModule --class=HeadingBlock --label=headingblock --plugin-id=headingblock --inputs='"name":"gender","type":"radio","label":"Gender","options":"array(\"male\" => \"Male\", \"female\" => \"Female\")"'

### Solution
Pipe separated options should be allowed. In block.html.twig template, options are rendered as it is, however they should be processed and an array should be generated.

**Comment by LOBsTerr** [comment]
@viveksr2 yes, you are right. I think pipe solution can help here and rest can be done in the twig. We just need to adapt the parser method

**Comment by enzolutions** [comment]
This fix was solved with https://github.com/hechoendrupal/drupal-console/pull/4032, but,  will be included in Drupal Console 2.0 because of breaks compatibility with previews releases 1.x.
